frontend/src/app/modules/intervention-form/helpers/option.ts

```diff
@@ -0,0 +1,13 @@
+interface SelectOption<T> {
+  value: string;
+  displayValue: T | string;
+}
+
+export class Option {
+  static of<T = string>(value: string, displayValue?: any): SelectOption<T> {
```

Comment:
I think `displayValue` should be a string, not `any`

Reply:
Good catch!